src/elements/content-preview/ContentPreview.js

```diff
@@ -384,6 +392,14 @@ class ContentPreview extends React.PureComponent<Props, State> {
         if (haveExperiencesChanged && this.preview && this.preview.updateExperiences) {
             this.preview.updateExperiences(previewExperiences);
         }
+
+        if (advancedContentInsights) {
+            const { advancedContentInsights: prevContentInsightsOptions } = prevProps;
+            const haveContentInsightsChanged = !isEqual(prevContentInsightsOptions, advancedContentInsights);
+            if (haveContentInsightsChanged && this.preview && this.preview.updateContentInsightsOptions) {
+                this.preview.updateContentInsightsOptions(advancedContentInsights);
```

Comment:
do we want to call this if `advancedContentInsights` updates from the previous config to `null` or `undefined`?

Reply:
No @tjuanitas . It won't be a possible scenario at this moment so I would left it as-is.